SchemaCreation/Indexes.sql

```diff
@@ -0,0 +1 @@
+CREATE NONCLUSTERED INDEX Loans_BorrowerID_BookID_Idx
ON Loans (BorrowerID,BookID)

CREATE NONCLUSTERED INDEX Loans_BookID_DateBorrowed_Idx
ON Loans (BookID,DateBorrowed)

CREATE NONCLUSTERED INDEX Books_BookID_Genre_Idx
ON Books (BookID,Genre)

CREATE NONCLUSTERED INDEX Loans_BookID_DateReturned_Idx 
ON Loans (BookID,DateReturned)

CREATE NONCLUSTERED INDEX Loans_DueDate_DateReturned_Idx 
ON Loans (DueDate, DateReturned)
```

Comment:
note: u can add index in table definition like this
CREATE TABLE Employees (
    EmployeeID INT PRIMARY KEY,
    DepartmentID INT,
    INDEX IX_DepartmentID (DepartmentID)
);